### PR TITLE
Add the service name to the log entry

### PR DESCRIPTION
**Pull request by git-hulk**
It's difficult to identify logs were generated by which service,
so we add the service name to help us filter faster.

**What changed?**

Add the service name to the log entry

**Why?**

It's difficult to identify logs were generated by which service,
so we add the service name to help us filter faster.

**How did you test it?**

Check the log entry whether has the right service name or not

**Potential risks**

No

### CODE REVIEW

File: cmd/server/cadence/server.go [hunk 1]
```diff
@@ -108,9 +108,10 @@ func (s *server) startService() common.Daemon {
 	params := service.BootstrapParams{}
 	params.Name = "cadence-" + s.name
 	params.Logger = loggerimpl.NewLogger(s.cfg.Log.NewZapLogger())
+	params.UpdateLoggerWithServiceName(s.name)
```
Comment (vytautas-karpavicius) [inline cmd/server/cadence/server.go:111]: In general this makes a lot of sense.

However this is not the only place where `service.BootstrapParams` parameters are constructed. There are:
- `host/onebox.go` used for integration tests. It needs updating as well.
- We also construct these params on the internal "Uber" version of Cadence. And we need to make sure to not forget adding this tag on the next release.

Comment (git-hulk) [reply cmd/server/cadence/server.go]: Thanks for your review, I have also added the `UpdateLoggerWithServiceName` to other places in onebox.

**Comment by git-hulk** [comment]
hi @vytautas-karpavicius How do you think about PR?

**Review by vytautas-karpavicius — APPROVED** [review]
Looks good now!

**Pull request merged**